### PR TITLE
Helper: Run shfmt

### PR DESCRIPTION
**Pull request by 1480c1**
going to test to make none of this is breaking

### CODE REVIEW

**Comment by ImportTaste** [comment]
Tested this on my (pre-existing) media-autobuild_suite. It went smoothly, no issues of any kind.

**Comment by wiiaboo** [comment]
Can you split the changes by type/category? Like only = to == in a single commit, then indentation only, etc.?

File: build/media-suite_helper.sh [hunk 1]
```diff
@@ -425,15 +425,16 @@ do_extract() {
...
-        { [[ $build32 = "yes" ]] && { [[ ! -f "$dirName"/build_successful32bit ]] ||
-            [[ -n "$flavor" && ! -f "$dirName/build_successful32bit_${flavor}" ]]; } ||
-          [[ $build64 = "yes" ]] && { [[ ! -f "$dirName"/build_successful64bit ]] ||
-            [[ -n "$flavor" && ! -f "$dirName/build_successful64bit_${flavor}" ]]; };
+    [[ -z $dirName ]] && dirName=$(guess_dirname "$archive")
+    if [[ $dirName != "." && -d $dirName ]] &&
+        {
+            [[ $build32 == "yes" ]] && { [[ ! -f "$dirName"/build_successful32bit ]] ||
+                [[ -n $flavor && ! -f "$dirName/build_successful32bit_${flavor}" ]]; } ||
+                [[ $build64 == "yes" ]] && { [[ ! -f "$dirName"/build_successful64bit ]] ||
```
Comment (wiiaboo) [inline build/media-suite_helper.sh]: This doesn't seem right. Isn't this line supposed to have one less indent?

Comment (1480c1) [reply build/media-suite_helper.sh]: I think it indented every time it hit `||`

**Comment by 1480c1** [comment]
wait, that's still wrong.

**Comment by wiiaboo** [comment]
You could also duplicate the rm -rf $dirName if it would make the if condition clearer.

File: build/media-suite_helper.sh [hunk 2]
```diff
@@ -429,19 +429,17 @@ do_extract() {
...
-    elif [[ -d $dirName ]]; then
+    if [[ $dirName != "." && -d $dirName ]]; then
+        if [[ $build32 == "yes" ]] &&
+            [[ ! -f "$dirName"/build_successful32bit || -n $flavor && ! -f "$dirName/build_successful32bit_${flavor}" ]]; then
+            rm -rf "$dirName"
+        elif [[ $build64 == "yes" ]] &&
+            [[ ! -f "$dirName"/build_successful64bit || -n $flavor && ! -f "$dirName/build_successful64bit_${flavor}" ]]; then
+            rm -rf "$dirName"
+        fi
+    elif [[ -d $dirName ]] || ! expr "$archive" : '.\+\(tar\(\.\(gz\|bz2\|xz\)\)\?\|7z\|zip\)$' > /dev/null; then
```
Comment (wiiaboo) [inline build/media-suite_helper.sh]: The expr part does not do the same thing. If it's not an archive, it should not try to enter a non-existing dirName

**Pull request merged**